### ISSUE TITLE
Figure out what Orthology data to load

### ISSUE THREAD

**Issue by zfinannee** [alliance-genome/agr_prototype#35]
The Orthology Working Group recommended that data be pulled directly from PANTHER.  The Portal Use Case Working Group (PUC WG) has some questions about what data exactly to pull, and so we would like to meet with them. 

This needs to be decided before Travis and Pedro can implement this part of the Use Case.

Using Label "question" for now, although this is question for the curators rather than the devs.

**Comment by chris-grove** [comment]
@zfinannee  I spoke to WormBase members of the orthology working group this morning and was basically told that we should just push ahead with PANTHER subfamilies, at least as a start for the portal. I will contact PANTHER to see if we can get a file with all PANTHER subfamilies and their gene members.

**Comment by chris-grove** [comment]
I actually found this FTP file which I think can give us everything we need, once we have unambiguous mappings to UniProt protein IDs for each gene:
ftp://ftp.pantherdb.org//sequence_classifications/current_release/PANTHER_Sequence_Classification_files/

**Comment by chris-grove** [comment]
I've created a folder in the Portal Data Google drive folder for Panther family data for each AGR species:

https://drive.google.com/drive/u/1/folders/0BxPsNle2cGvPZndJSGtCYzZLbGs

**Comment by gabinkley** [comment]
Pedro is indexing the Panther data in https://drive.google.com/drive/u/0/folders/0BxPsNle2cGvPZndJSGtCYzZLbGs
and is calling it 'Homology Groups' instead of 'Orthology Groups' 

If this is NOT OK, then speak up.  Once again, this is to show functionality, even if the data is not 100% correct.

**Comment by chris-grove** [comment]
@gabinkley @pedrohr 
Yep, sounds good! Thanks!

**Comment by chris-grove** [comment]
@gabinkley @pedrohr 
We decided on yesterday's Use Case working group call that we would abandon the PANTHER families/subfamilies as groups of genes to display. Instead we will work on developing our next use case which will be a gene-centric page in AGR that will focus on a single gene and display data for it and all of its orthologs in other species and paralogs in the same species. As @khowe mentioned above, these pairwise orthology calls can be taken from PANTHER here:

ftp://ftp.pantherdb.org/ortholog/current_release/RefGenomeOrthologs.tar.gz

So, bottom line is that the developers shouldn't spend more effort trying to import PANTHER subfamilies for the first use case and prototype. We will outline what info we think should be displayed on an AGR gene-centric page in our next use case.

**Comment by gabinkley** [comment]
Closing.  This decision will be implemented.
